example/App.tsx

```diff
@@ -134,21 +149,43 @@ function App(): React.JSX.Element {
   }, [users, isTrue]);
 
   useEffect(() => {
-    if (apiKey) {
       const filterStateMessage = [
         ConversationState.MEMBER_LEFT_SECRET_CHATROOM,
       ]; // give type of conversation to be filtered using ConversationState enum
 
       // proivde apiKey below to initMyClient
-      const res: any = initMyClient(apiKey, filterStateMessage); // pass api key as first param and filterStateMessage array as second
+      const res: any = initMyClient(filterStateMessage); // pass api key as first param and filterStateMessage array as second
       setMyClient(res);
-    }
   }, [isTrue, apiKey]);
 
   useEffect(() => {
     setStyles();
   }, []);
 
+  const callbackClass = new LMCoreCallbacks(
+    (a: string, b: string) => {
```

Comment:
rename to `accessToken` and `refreshToken`\
```suggestion
    (accessToken: string, refreshToken: string) => {
```